Docs: add note forbidding allowance of user input of JS (#1394)
grafana/plugin-tools: docusaurus/docs/get-started/best-practices.md

- **Include default dashboards in your data source or app plugin** - Refer to [Bundling of dashboards](../key-concepts/plugin-types-usage.md#bundling-of-dashboards) for more information.
- **Ensure that the minimum version of Grafana is correct** - Make sure that the `grafanaDependency` in your `plugin.json` points to the earliest version of Grafana that your plugin fully supports.
- **Don't expose sensitive information** - For security reasons, avoid exposing sensitive information such as secrets. Make sure to utilize log levels properly, avoid excessive logging, and never log credentials or other sensitive information.

> - **Don't allow user input of JavaScript** - For security reasons, don't let your plugin's users enter arbitrary JavaScript or other potentially malicious code.

Reviewer: Can we rewrite this to something in the line of.

```suggestion
- **Don't allow user input of arbitrary code** - For security reasons, don't let your plugin's users enter potentially malicious code to prevent arbitrary code execution.
```

Reviewer 2: nice! this also can be a problem for backend plugins that would like to execute user-input code.
